src/h5web/providers/mock/data.ts

```diff
@@ -252,6 +265,7 @@ export const mockMetadata = makeMetadata({
       makeDatasetHardLink('scalar_str'),
     ]),
     makeGroup('nD_datasets', undefined, [
+      makeDatasetHardLink('oneD_simple'),
```

Comment:
I added a new simple 1D dataset `y=x` for the LineVis story. The dataset `oneD` is very nice for other stories but not for the log scale stories of the LineVis.

Reply:
Is this the plot you don't like?

![image](https://user-images.githubusercontent.com/2936402/98118983-45409280-1eac-11eb-861d-ef31926c8770.png)

Right, so you want `y=x + a` basically. Makes sense. Not sure about the term `simple`, though, since we use it to refer to HDF5 simple shapes. Perhaps `oneD_linear`?

Reply:
This one and the `Negative Log Scale` that doesn't show anything given that there are only positive values. :+1: for `oneD_linear`